dashboard/test/controllers/api/v1/pd/workshop_summary_report_controller_test.rb

```diff
@@ -192,6 +192,21 @@ class Api::V1::Pd::WorkshopSummaryReportControllerTest < ::ActionController::Tes
     assert_equal EXPECTED_COMMON_FIELDS.count + EXPECTED_PAYMENT_FIELDS.count, response.first.count
   end
 
+  test 'includes unpaid workshops' do
+    unpaid_workshop = create :pd_ended_workshop, organizer: @organizer, course: Pd::Workshop::COURSE_CSD
+    create :pd_workshop_participant, workshop: @workshop, enrolled: true, in_section: true, attended: true
+
+    sign_in @admin
+    get :index
+    assert_response :success
+    response = JSON.parse(@response.body)
+    assert_equal 3, response.count
+    unpaid_report = response.find{|row| row['workshop_id'] == unpaid_workshop.id}
```

Comment:
@bcjordan It's from https://github.com/code-dot-org/code-dot-org/pull/12651. There's an open Hound [issue](https://github.com/houndci/hound/issues/1290) that it's not loading included rubocop config files. This passes linting with our local rules.